pa11y/.pa11yci

```diff
@@ -24,6 +24,19 @@
       {
          "url": "https://digideps.local",
          "screenCapture": "screenshots/index.png"
+      },
+      {
+         "url": "https://digideps.local/report/{id}/overview",
```

Comment:
I believe the {id} placeholder won't work and you are required to be the actual id of the report in.

Reply:
You're right but unfortunately, we're never going to know the exact id of a report to view when this runs in the pipeline. I thought it was better to have a good description of the page we're testing and then manually navigate to the login page/further pages in the pa11y actions.